CodeSnippetsReflection.OpenAPI/LanguageGenerators/GraphCliGenerator.cs

```diff
@@ -225,9 +226,10 @@ private static void ProcessMeSegments(List<string> commandSegments, string opera
             {
                 string pattern = "\\?\\$\\w*=";
                 string[] splittedQueryString = Regex.Split(snippetModel.QueryString, pattern, RegexOptions.Compiled, TimeSpan.FromSeconds(5));
+                string queryOptionFunction = IsUrlEncoded(splittedQueryString[1]) ? HttpUtility.UrlDecode(splittedQueryString[1]) : splittedQueryString[1];
```

Comment:
```suggestion
                string queryOptionFunction = HttpUtility.UrlDecode(splittedQueryString[1]);
```

I would suggest dropping the `IsUrlEncoded` function as `HttpUtility.UrlDecode` is always called.(to check if the url is decoded and again if its encoded)  and it will return the same string if the url is not decoded making the branch redundant.

Reply:
Yes you're right. Thanks